runtime/onert/core/src/ir/pass/ConstantInsertionPass.cc

```diff
@@ -35,7 +35,7 @@ void ConstantInsertionPass::callback(const OperationIndex &node_index, Operation
   const auto layout = op_seq_lower_info->layout();
   const auto factor = operand::PermuteFactor{backend, layout};
 
-  for (const auto input : node.getInputs().asUnique())
+  for (const auto input : node.getInputs() | Remove::duplicated | Remove::optional)
```

Comment:
`Remove::optional` filter is probably not needed here.

Reply:
@ragmani Thank you for review. I will remove it. It was an example for showing consecutive filters. I would like to get your review for #1441, in which, I added `Remove::optional` mechanically for almost all iterations of getInputs().